### PR TITLE
DEX-810 upgrade Uppy to version 2

### PR DESCRIPTION
**Pull request by Emily-Ke**
Upgrade Uppy to the latest version, version 2, in order to get access to a `limit` option with a default. The upgrade forced the usage of the "new" keyword.

### CODE REVIEW

File: src/hooks/useReportUppyInstance.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default function useReportUppyInstance(reportType) {
   fileRef.current = files;
 
   useEffect(() => {
-    const uppyInstance = Uppy({
+    const uppyInstance = new Uppy({
```
Comment (Atticus29) [inline src/hooks/useReportUppyInstance.js:19]: Uppy instances also appear in ProfileUploader.jsx ln.20 , UploadManager.jsx ln 32, SettingsFileUpload.jsx ln. 39,

**Review by Atticus29 — CHANGES_REQUESTED** [review]
Might need a few more "new"s (see comment). Otherwise, LGTM

File: src/hooks/useReportUppyInstance.js (hunk 1 above)
Comment (Emily-Ke) [reply src/hooks/useReportUppyInstance.js:19]: Thanks!

**Review by brmscheiner — APPROVED** [review]

**Review by Atticus29 — APPROVED** [review]
Looks good!

**Pull request merged**